utils/api.py

```diff
@@ -79,21 +79,22 @@ def tasks_create_file():
     platform = request.forms.get("platform", "")
     tags = request.forms.get("tags", None)
     custom = request.forms.get("custom", "")
-    memory = request.forms.get("memory", False)
+    memory = request.forms.get("memory", 'False')
     clock = request.forms.get("clock", None)
     shrike_url = request.forms.get("shrike_url", None)
     shrike_msg = request.forms.get("shrike_msg", None)
     shrike_sid = request.forms.get("shrike_sid", None)
     shrike_refer = request.forms.get("shrike_refer", None)
-    
-    if memory.isdigit() and int(memory):
-        memory = True
-    elif memory:
+
```

Comment:
BTW, you can't do the check in elif because elif memory of anything other than when memory is None will be true.  'False" will be true.  I tested it.

```
Python 2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> True.isdigit()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'bool' object has no attribute 'isdigit'
>>> 'True'.isdigit()
False
>>> bool('True')
True
>>> bool('False')
True
```